pcfuncs/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM mcr.microsoft.com/azure-functions/python:4-python3.8
+FROM mcr.microsoft.com/azure-functions/python:4-python3.10
```

Comment:
Revert to 3.8 after removing redundant packages in pcfuncs/requirements.txt since they are declared in pccommon already which caused some dependency conflicts previously.